tests/pipelines/controlnet/test_controlnet_inpaint_sdxl.py

```diff
@@ -57,6 +57,7 @@ class ControlNetPipelineSDXLFastTests(
     image_latents_params = TEXT_TO_IMAGE_IMAGE_PARAMS
 
     def get_dummy_components(self):
+        torch.manual_seed(0)
```

Comment:
Otherwise for consecutive runs, the underlying pipeline will yield different results.